src/main/scala/no/ndla/searchapi/service/search/MultiDraftSearchService.scala

```diff
@@ -181,16 +181,18 @@ trait MultiDraftSearchService {
       ).flatten
     }
 
-    private def draftStatusFilter(statuses: Seq[draft.ArticleStatus.Value]) =
+    private def draftStatusFilter(statuses: Seq[draft.ArticleStatus.Value]): Some[BoolQuery] = {
+      val draftStatuses = Seq("draftStatus.current", "draftStatus.other")
       if (statuses.isEmpty) {
         Some(
-          boolQuery.not(termQuery("draftStatus", ArticleStatus.ARCHIVED.toString))
+          boolQuery.not(draftStatuses.map(s => termQuery(s, ArticleStatus.ARCHIVED.toString)))
```

Comment:
Her vil statusen ARCHIVED kun kunne eksistere i current så denne kunne vore gjort nesten identisk som gamle versjonen. Men du kan bestemme om du vil ta hensyn til denne kommentaren.
```suggestion
          boolQuery.not(termQuery("draftStatus.current", ArticleStatus.ARCHIVED.toString))
```